### PR TITLE
Adding styles & Configurations for R3

### PR DESCRIPTION
**Pull request by hanbyul-here**
**Related Ticket:** #1098 
+ We are encouraging IMPACT team to use USWDS. This new style configuration that uses grid-layout will be useful for IMPACT team. (I also need this style for Carousel )

### Description of Changes
- Added grid-layout style from uswds design system
- Edited purgecss configuration to include `instance/overrides` folder as content
- Added CSS Variables for VEDA customization colors. 

### Notes & Questions About Changes

I almost gave up getting purgecss to work, but @dzole0311 helped me figure out what was wrong. (The default extractor doesn't allow special characters like `:`, which we need to target specific screen size like `tablet: ` ex. https://designsystem.digital.gov/utilities/layout-grid/ Thank you so much 🙇 @dzole0311 !

@slesaad , you don't need to review this PR, but I think it would be beneficial to check how we are importing the styles and purging the styles through `postcss.config`.

### Validation / Testing
This PR doesn't bring any visual change to UI, but this GHG pr uses this branch: https://github.com/US-GHG-Center/veda-config-ghg/pull/482

### CODE REVIEW

**Comment by hanbyul-here** [comment]
purgecss configuration : https://github.com/FullHuman/purgecss/issues/86#issuecomment-391279202

**Review by dzole0311 — APPROVED** [review]
Thank you @hanbyul-here! Looking forward to any future feedback on USWDS usage from @slesaad as well as we continue to transition towards USWDS.

In addition to the docs mentioned in the description, we also have on our disposal the `<Grid />` React component from the react implementation of the USWDS library for layout purposes (depends on preference of the developer). Link to the relevant docs: https://trussworks.github.io/react-uswds/?path=/docs/components-grid--docs.

**Comment by hanbyul-here** [comment]
Oh!! Thanks a lot for pointing the grid component from react-uswds @dzole0311 ! I didn't even think about it 😓 We probably need to list grid-related classes as a whitelist to use it, though, I guess?

File: postcss.config.js [hunk 1]
```diff
@@ -16,32 +18,35 @@ let reactCalendarPath = path.resolve(
...
   );
   reactCalendarPath = path.resolve(
-    basePath,
+    vedaUIPath,
     'node_modules/react-calendar/dist/**/*.css'
   );
+
+  contentPaths = [
+    `${vedaUIPath}/app/**/*.{js,jsx,ts,tsx, html}`,
+    `${basePath}/overrides/**/*.{js,jsx,ts,tsx}`
```
Comment (slesaad) [inline postcss.config.js:33]: should this also include `custom-pages`? (sorry if I'm misunderstanding this :D )

Comment (hanbyul-here) [reply postcss.config.js:33]: Yes, I forgot that there are also custom pages; I will add them. Thanks for catching it!

**Pull request merged**